File: easy/invert-binary-tree/invert-binary-tree.py
class Solution:
    def invertTree(self, root: Optional[TreeNode]) -> Optional[TreeNode]:
        def dfsinvert(tree):
            if tree != None:
                [tree.right, tree.left] = [tree.left, tree.right]
                dfsinvert(tree.left)
                dfsinvert(tree.right)
        
        dfsinvert(root)
        return root

File: easy/invert-binary-tree/invert-binary-tree.java
class Solution {
    protected void dfs(TreeNode root) {
        if (root!=null) {
            TreeNode temp=root.left;
            root.left=root.right;
            root.right=temp;
            dfs(root.left);
            dfs(root.right);
        }
    }
    public TreeNode invertTree(TreeNode root) {
        dfs(root);
        return root;
    }
}
